README.md

```diff
@@ -21,24 +21,31 @@ A cloudinary driver for facades.Storage of Goravel.
         &cloudinary.ServiceProvider{},
     }
     ```
-3. Add cloudinary disk to `config/filesystems.go` file
-   ```go
-   // config/filesystems.go
-   import (
-         cloudinaryFacades "github.com/goravel/cloudinary"
-         "github.com/goravel/framework/filesystem"
-   )
+   3. Add cloudinary disk to `config/filesystems.go` file
+      ```go
+      // config/filesystems.go
+      ...
    
-   "disks": map[string]filesystem.Disk{
-         ...
-         "cloudinary": map[string]any{
-               "driver": "custom",
-               "cloud": "your_cloud_name",
-               "key": "your_api_key", 
-               "secret": "your_api_secret",
-               "via": func()(filestystem.Disk, error) {
-                     return cloudinaryFacades.Cloudinary("cloudinary"), nil
-               },
-         }
-   }
-   ```
\ No newline at end of file
+      import (
+            cloudinaryfacades "github.com/goravel/cloudinary/facades"
+            "github.com/goravel/framework/filesystem"
+      )
+   
+      "disks": map[string]filesystem.Disk{
+            ...
+            "cloudinary": map[string]any{
+                  "driver": "custom",
+                  "cloud":  config.Env("CLOUDINARY_CLOUD"),
+                  "key":    config.Env("CLOUDINARY_ACCESS_KEY_ID"), 
+                  "secret": config.Env("CLOUDINARY_ACCESS_KEY_SECRET"),
+                  "resource_types": map[string][]string{
```

Comment:
S: Could you please add a comment for this? To remind the configuration is optional and its action.